Source/Shaders/GroundPolylineBatchVS.glsl

```diff
@@ -0,0 +1,21 @@
+attribute vec3 currentPosition;
+attribute vec3 previousPosition;
+attribute vec3 nextPosition;
+attribute vec2 expandAndWidth;
+attribute float a_batchId;
+
+uniform mat4 u_modifiedModelView;
+
+void main()
+{
+    float expandDir = expandAndWidth.x;
+    float width = abs(expandAndWidth.y) + 0.5;
+    bool usePrev = expandAndWidth.y < 0.0;
+
+    vec4 p = u_modifiedModelView * vec4(currentPosition, 1.0);
```

Comment:
If you don't do the above, still declare them as `vec4` and `w` will be implicitly `1.0`, which will clean this up.